docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -1,5 +1,8 @@
 ---
-description: Avalanche is a consensus protocol that is scalable, robust, and decentralized.
+tags: [consensus, snowball, protocol]
```

Comment:
Tags are visible links embed at the bottom of each page and should only include the domains the page lives under. Words like this should be in the `keywords` metadata field. See [here](https://github.com/ava-labs/avalanche-docs/blob/master/docs/build/subnet/deploy/custom-vm-subnet.md) for a good example.

Reply:
Gotcha ✅